WordPress/gutenberg
# Scripts: Fix CSS imports not minified

Runs PostCSS on the imported files.

Closes #55885.



## What?

Allows CSS imports to be minified.

## Why?
The files produced by npm run build should be minified.

## How?
The webpack config is adjusted to use PostCSS on imported files also.

## Testing Instructions
Use custom webpack.config.js and set an entry for a stylesheet.
In the stylesheet, add an import to get the content of another stylesheet.
Run the build script.

You shoud get a dist file with minified content.
Without this fix, the content that is imported is not minified.

See https://github.com/WordPress/gutenberg/issues/55885

### Testing Instructions for Keyboard
